src/js/createSpecialButton/outputValueKeydown.js

```diff
@@ -0,0 +1,30 @@
+//  highlight keys and value output
+export function outputValueKeydown(array, specialButton, textareaElement) {
+  document.addEventListener('keydown', (event) => {
+    array.map((buttonKeyboard, i) => {
+      if (event.code === buttonKeyboard[2]) {
+        if (i === 13 || i === 14  || i === 28 || i === 41 || i === 42  || i === 54 || i === 58 ){
```

Comment:
magic numbers here, i can't understand what they do. you should move them to constants: 
`const SHIFT_KEY = number`